osmose_config.py

```diff
@@ -1208,7 +1208,7 @@ class gen(default_country):
 default_country("asia", "south_korea", 307756, {"country": "KR", "language": "ko", "proj": 32652}, download_country="south-korea")
 default_country("asia", "syria", 184840, {"country": "SY", "language": "ar", "proj": 32637})
 default_country("asia", "tajikistan", 214626, {"country": "TJ", "language": "tg", "proj": 32642})
-default_country("asia", "taiwan", 3777248, {"country": "TW", "language": ["zh_TW", "en"], "proj": 32651}, download_repo=GEOFABRIK)
+default_country("asia", "taiwan", 449220, {"country": "TW", "language": ["zh_TW", "en"], "proj": 32651}, download_repo=GEOFABRIK)
```

Comment:
Did you see that this relation includes some disputed islands near China's coast, that might be in another region analysed by osmose?

Reply:
All Taiwan is disputed.

But this costal island are excluded on mainland China https://www.openstreetmap.org/relation/553303#map=7/25.488/119.399

Reply:
In fact, this extract comes from Geofabrik, and is almost correct: http://download.geofabrik.de/asia/taiwan.html  (some maritime borders are missing)

Reply:
On Geofabrik poly there is two missing areas on south.
https://www.openstreetmap.org/relation/449220#map=6/16.289/118.059